tests/data/expected/main/jsonschema/custom_template_relative_api.py

```
@@ -0,0 +1,12 @@
+# generated by datamodel-codegen:
+#   filename:  pet_simple.json
+#   timestamp: 1985-10-26T08:21:00+00:00
+
+from __future__ import annotations
+
+from pydantic import BaseModel
+
+
+class Pet(BaseModel):
+    template_revision: str = 'caller'
+    include_revision: str = 'caller'
```

Ruff (0.16.3)

[warning] No newline at end of file

Add trailing newline

(W292)